CorsixTH/Src/th_lua.h

```diff
@@ -408,6 +427,10 @@ void luaT_execute(lua_State* L, const char* sLuaString, T1 arg1, T2 arg2,
 
 void luaT_pushtablebool(lua_State* L, const char* k, bool v);
 
+// Print debugging helper functions
+
+void luaT_printvalue(lua_State* L, int idx);
+
```

Comment:
Ah, probably this function thus, seems fine then.

I fail to see the point of having an empty line between the function declarations though. Double-spacing code doesn't make it simpler imho.